scripts/figma-import/icons.js

```diff
@@ -0,0 +1,81 @@
+const config = require('../../figma-import.config')
+
+const path = require('path')
+const fs = require('fs').promises
+const { pascalCase } = require('change-case')
+
+const { getImageContent, getNodeChildren, getSvgImageUrl } = require('./utils/api')
+
+const IconsDir = path.resolve(__dirname, '../../src/shared/icons/figmaSvgs')
+
+const getIconFolderPath = () => path.resolve(IconsDir)
+
+let counter = 0
+
+/**
+ * clear icons dir
+ *
+ */
+const clearIconsDir = async () => {
+  try {
+    await fs.rmdir(IconsDir, { recursive: true })
```

Comment:
Im getting this in the console: 
```
(node:22653) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead
```
This is probably due the fact that, Im using node 16, but I think ` fs.rm(IconsDir, { recursive: true })` should also works on node 14 and we can safely replace it.